### DISCUSSION TITLE
2012/04/quick-tip-always-show-solution-in-visual-studio-2010/

### DISCUSSION THREAD

**Discussion by giscus** [SeanKilleen/seankilleen.github.io#802] [Specific Post Comments]
# 2012/04/quick-tip-always-show-solution-in-visual-studio-2010/

Problem:

https://seankilleen.com/2012/04/quick-tip-always-show-solution-in-visual-studio-2010/

**Comment by GrumpyOldDBA** [comment]
Seems that the option has been removed but effectively true.
See https://stackoverflow.com/a/58574459/1585164
